packages/portal/src/components/stories/StoriesTagsDropdown.vue

```diff
@@ -9,7 +9,10 @@
     />
     <div
       ref="tagsdropdown"
+      v-click-outside="clickOutsideConfig"
       class="position-relative mb-4"
+      @focusin="handleFocusin"
```

Comment:
Why is the focus event handler moved up to this div? Shouldn't it stay on the input as that's the element receiving focus?

Reply:
It was to try and make sure the click outside config was always active when either the input or the dropdown are in focus. 
It's not possible to open the dropdown without focusing on the input first now though, so yeah I've moved the handler to the input.
This way there are less events being fired when moving focus around inside the dropdown. 👍
It still re-activates the config when tabbing back to the input, but at least it's not activating on just navigating within the dropdown.